test/distributed/fsdp/test_fsdp_hybrid_shard.py

```diff
@@ -97,67 +97,58 @@ def test_raises_manual_wrap_hybrid_shard_when_none_policy(self):
         with err_ctx:
             model = FSDP(model, sharding_strategy=ShardingStrategy._HYBRID_SHARD_ZERO2)
 
-
-    @skip_if_lt_x_gpu(2)
-    def test_hybrid_shard_strategy_mismatch_raises(self):
-        for sharding_strategy in [
-            ShardingStrategy._HYBRID_SHARD_ZERO2,
-            ShardingStrategy.HYBRID_SHARD
-        ]:
-            with self.subTest(sharding_strategy=sharding_strategy):
-                model = MyModel().cuda()
-                intra_pg = self.process_group
-                inter_pg = dist.new_group(ranks=[self.rank])
-                model.lin1 = FSDP(model.lin1, process_group=(intra_pg, inter_pg), sharding_strategy=sharding_strategy)
-                self.assertEqual(model.lin1.process_group, intra_pg)
-                self.assertEqual(model.lin1._inter_node_pg, inter_pg)
-                model = FSDP(model, process_group=intra_pg)
-                inp = torch.randn(4, 10)
-                # Errors during _lazy_init
-                with self.assertRaisesRegex(ValueError, "expect sharding strategies to be the same"):
-                    model(inp)
-
     @skip_if_lt_x_gpu(2)
     def test_hybrid_shard_pg_mismatch_raises(self):
         model = MyModel().cuda()
         intra_pg = self.process_group
         inter_pg = dist.new_group(ranks=[self.rank])
         # Mismatched process groups for intra-node
         model.lin1 = FSDP(
-            model.lin1, process_group=(intra_pg, inter_pg), sharding_strategy=ShardingStrategy.HYBRID_SHARD
+            model.lin1,
+            process_group=(intra_pg, inter_pg),
+            sharding_strategy=ShardingStrategy.HYBRID_SHARD,
         )
         model = FSDP(
-            model, process_group=(dist.new_group(), dist.new_group()), sharding_strategy=ShardingStrategy.HYBRID_SHARD
+            model,
+            process_group=(dist.new_group(), dist.new_group()),
+            sharding_strategy=ShardingStrategy.HYBRID_SHARD,
         )
         # Errors during _lazy_init
         inp = torch.randn(4, 10)
-        with self.assertRaisesRegex(ValueError, "intra-node process groups do not match"):
+        with self.assertRaisesRegex(
+            ValueError, "intra-node process groups do not match"
+        ):
             model(inp)
 
         # Mismatched process groups for inter-node
         model = MyModel().cuda()
         model.lin1 = FSDP(
-            model.lin1, process_group=(intra_pg, inter_pg), sharding_strategy=ShardingStrategy.HYBRID_SHARD
+            model.lin1,
+            process_group=(intra_pg, inter_pg),
+            sharding_strategy=ShardingStrategy.HYBRID_SHARD,
         )
         model = FSDP(
-            model, process_group=(intra_pg, dist.new_group()), sharding_strategy=ShardingStrategy.HYBRID_SHARD
+            model,
+            process_group=(intra_pg, dist.new_group()),
+            sharding_strategy=ShardingStrategy.HYBRID_SHARD,
```

Comment:
nit: these formatting changes are another example where reviewers may need to read carefully to check if there are any logical changes to the test.

Reply:
Makes sense.